old/tools.ts

```diff
@@ -0,0 +1,135 @@
+const chooseOne = <T>(list: T[]): T => {
```

Comment:
You should be able to remove this file and its containing folder as they won't ever be called. `old` should be able to be totally removed now.

Reply:
great!